nservicebus/messaging/send-a-message.md

```diff
@@ -1,43 +1,54 @@
 ---
-title: Sending a Message
-summary: Describes how to send a message
+title: Sending messages
+summary: Describes how to send messages
 tags: []
 redirects:
 - nservicebus/how-do-i-send-a-message
 ---
 
-Message sending involves using Send operation that takes an argument of a message instance to be delivered. The details differ slightly between version of NServiceBus, as shown in following snippets.
+NServiceBus supports sending different types of messages (see [Messages, Events and Commands](messages-events-commands.md)) to any endpoint. When a message arrives at an endpoint it goes through a pipeline of processing steps. Messages can be sent from outside or inside the message processing pipeline:
 
-Here's how a message is sent by directly using the instance of the endpoint
+## Outside the pipeline
```

Comment:
while I agree that talking about the handler is certainly easier to understand than the pipeline, but it's also plain wrong (behaviors or other extension points) so I hesitate.